nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -98,9 +98,10 @@ In addition to numerous new and upgraded packages, this release includes the fol
   and [changelog](https://ngrok.com/docs/ngrok-agent/changelog). Notably, breaking changes are that the config file format has
   changed and support for single hyphen arguments was dropped.
 
-- `i18n.supportedLocales` is now only generated with the locales set in `i18n.defaultLocale` and `i18n.extraLocaleSettings`.
-  - This reduces the final system closure size by up to 200MB.
-  - If you require all locales installed, set the option to ``[ "all" ]``.
+- locales generated through `i18n.supportedLocales` now only contain the locales set in `i18n.defaultLocale` and `i18n.extraLocaleSettings`.
+  - Previously by default all locales where generated which used about 200MB.
+  - If you set a different locale in your desktop environment or `LC_*` variables then defined in the above listed options, you need to adjust your config to avoid breakages.
```

Comment:
typo: then → than

Maybe give a concrete suggestion like
``If you set a different locale in your desktop environment or `LC_*` variables than defined in the above listed options, you need to add the locale to `i18n.extraLocaleSettings`.``
?